services/core/methods/controllers/fees.js

```diff
@@ -24,6 +24,9 @@ const getEstimateFeeByte = async () => {
 	result.feeEstimatePerByte.low = response.low;
```

Comment:
Does it make sense to resolve  all properties for `result`  and `meta` at `core` level and pass it directly to controllers? (In that way, we can avoid adding any logic to controller)

For example: we can do something like  `const fees = {data: [], meta: {}}`  in `shared/core/dynamicFees`